docker-cloudwatch-exporter.tf

```diff
@@ -0,0 +1,66 @@
+resource "github_repository" "cloudwatch_exporter" {
+  name             = "cloudwatch-exporter"
```

Comment:
maybe 
```suggestion
  name             = "prometheus-cloudwatch-exporter"
```

Reply:
Don't disagree with this but we already have another prometheus application, alertmanager. That does not begin with prometheus and I would like them to be consistent whichever way we go. Are you attached to it beginning with prometheus?

Reply:
my concern is when we have to implement other CloudWatch exporters, like the one for Splunk